src/service/v2/oidc.go

```diff
@@ -85,8 +85,23 @@ func (o *OIDC) GetMe(ctx context.Context, session *domain.OIDCSession) (*service
 	return o.user.getMe(ctx, session)
 }
 
-func (o *OIDC) GetActiveUsers(ctx context.Context, session *domain.OIDCSession) ([]*service.UserInfo, error) {
-	return o.user.getActiveUsers(ctx, session)
+func (o *OIDC) GetActiveUsers(ctx context.Context, session *domain.OIDCSession, includeBot bool) ([]*service.UserInfo, error) {
+	users, err := o.user.getActiveUsers(ctx, session)
+	if err != nil {
+		return nil, err
+	}
+	if includeBot {
+		return users, nil
+	}
+	// includeBotがfalseの場合はbotユーザーを除外する
+	filteredUsers := make([]*service.UserInfo, 0, len(users))
+	for _, user := range users {
+		if !includeBot && user.GetBot() {
```

Comment:
ここの includeBot はfalseどうあることが確定しているので不用です
```suggestion
		if user.GetBot() {
```